src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs

```diff
@@ -183,6 +187,53 @@ private static bool IsForEachProperty(IPropertySymbol symbol)
             return locations.ToImmutableAndFree();
         }
 
+        private async Task<ImmutableArray<FinderLocation>> FindConditionalAccessReferencesAsync(
+            IPropertySymbol symbol, Document document, SemanticModel semanticModel,
+            FindReferencesSearchOptions options, CancellationToken cancellationToken)
```

Comment:
this looks to be a large copy of `FindElementAccessReferencesAsync`.  We should ideally share more code here.

Reply:
I made some changes to remove code duplications.